src/main/resources/i18n/messages.properties

```diff
@@ -692,6 +692,9 @@ project.nav.details=Recent Activity
 project.nav.settings=Settings
 project.nav.samples.import-metadata=Import Sample Metadata
 
+# Project select
+ProjectSelect.label.id=id: {0}
```

Comment:
Can you try using uppercase on the ID, I think it might look better.

Reply:
Please see ca4b83f8250a2e286e7063d4111267331eee23f3.